build.sbt

```diff
@@ -223,6 +223,32 @@ lazy val client = (project in file("target/clients/java"))
     }
   )
 
+lazy val pythonClient = (project in file("target/clients/python"))
+  .enablePlugins(OpenApiGeneratorPlugin)
+  .settings(
+    name := s"$artifactNamePrefix-client",
+    commonSettings,
+    (Compile / compile) := ((Compile / compile) dependsOn generate).value,
+
+    // OpenAPI generation specs
+    openApiInputSpec := (file(".") / "api" / "all.yaml").toString,
+    openApiGeneratorName := "python",
+    openApiOutputDir := (file("target") / "clients" / "python").toString,
+    openApiPackageName := s"$artifactNamePrefix",
```

Comment:
this should probably be $artifactNamePrefix_client?

Reply:
I've named it `$artifactNamePrefix` to match the current name of the [published package](https://pypi.org/project/unitycatalog/). Should we change it?

Reply:
I think it then makes sense to leave it as is. cc: @tdas